### PR TITLE
Add new empty state for Bots page in the web UI

### PR DESCRIPTION
**Pull request by avatus**
This will add a new "empty state" component to the Bots page when no bots exist yet.

Screenshot 2024-09-19 at 11 12 27 AM

### CODE REVIEW

File: web/packages/shared/components/EmptyState/EmptyState.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
+import styled from 'styled-components';
```
Comment (avatus) [inline web/packages/shared/components/EmptyState/EmptyState.tsx:1]: i may abstract more here and/or abstract away the logic for the automated slider into a hook as well, but I want to finish the device trust empty state to see whats needed first

**Comment by zmb3** [comment]
Something looks a tiny bit off with the bottom margins in storybook.

image
Screenshot 2024-09-19 at 11 44 35 AM

**Comment by avatus** [comment]
> Something looks a tiny bit off with the bottom margins in storybook.

Hmm, ok ty! I can see it when I make my screen smaller even outside storybook. will check it out, thank you

**Comment by avatus** [comment]
Was just an issue with a missing `px` after a font size. The margins now correctly encompass all the text at any size

**Review by zmb3 — APPROVED** [review]

File: web/packages/teleport/src/Bots/List/EmptyState/EmptyState.tsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    return () => clearInterval(id);
+  }, []);
+
+  return (
+    <Box mt={4} data-testid="bots-empty-state">
+      <Box mb={3}>
+        <H1 mb={3}>What are Bots?</H1>
+        <Text css={{ maxWidth }}>
+          Static keys and API keys in your automated workflows are the target of
+          hackers and are one of the key sources of security breaches.
```
Comment (strideynet) [inline web/packages/teleport/src/Bots/List/EmptyState/EmptyState.tsx:54]: Slight change to the copy here so we don't say "key" three times in one sentence might be nice 😓 

```suggestion
          hackers and are one of the primary sources of security breaches.
```

File: web/packages/teleport/src/Bots/List/EmptyState/EmptyState.tsx [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+          <br />
+          Teleport Machine ID replaces shared credentials and secrets with
+          short-lived x.509 or SSH certificates and gives you a unified plan to
+          register, define access policies, and audit all your workflows.
```
Comment (strideynet) [inline web/packages/teleport/src/Bots/List/EmptyState/EmptyState.tsx:58]: This sentence doesn't make much sense to me - "register, define" ?

**Review by strideynet — APPROVED** [review]

File: web/packages/teleport/src/Bots/List/EmptyState/EmptyState.tsx (hunk 3 above)
Comment (avatus) [reply web/packages/teleport/src/Bots/List/EmptyState/EmptyState.tsx:58]: I'm not sure (I didn't write the copy) but I read it as
gives you a unified place to
- register
- defined access policies
- and audit your workflows

register _what_ is probably the question?

Comment (strideynet) [reply web/packages/teleport/src/Bots/List/EmptyState/EmptyState.tsx:58]: > register what is probably the question?

Yup - that's exactly what I'm wondering ahhaha

**Pull request merged**